_data/monthly-calls.yml

```diff
@@ -1,5 +1,17 @@
 videos:
-  - title: A first look at U.S. Web Design System Web Components
```

Comment:
Hey @amyleadem , this doesn't look right to me, but when I go to edit, everything looks like it's in the right place. What am I doing wrong?

Reply:
Looks like there were some very minor spacing issues in the yaml file. YAML relies on spacing to know where data should live, so it can be a bit persnickety. I _believe_ that I resolved the issue in [5329b47](https://github.com/uswds/uswds-site/pull/2827/commits/5329b47edc8c2877fb791ced6102d7b82695bed6)

Reply:
⁠I thought I had that spacing thing on point. It's always the spacing with me!!! UGH. Thank you!!

Reply:
YAML is tricky. You're doing great!